### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards: Schema Explorer]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8164]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL in the browser.
2. Run the accessibility insight for web tool and observe the issue.

**Actual Result:**
Color contrast between foreground and background of 'Comment and View' button text is 4.2:1 which is less than 4.5:1.

**Expected Result:**
Color contrast ration between foreground and background of 'Comment and View' button text should be equal to greater than 4 .5:1.

**Note:**
Same issue is repro under **'Activity Update'** screen as well.

**User Impact:**
It will impact the low vision user if the contrast ratio of text is less than 4.5:1 as they will face difficulties to see the elements properly.

**Element path:** 
button[aria-label="View"] > div

**How to fix:** 
Fix any of the following:
  Element has insufficient color contrast of 4.41 (foreground color: #0079db, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1.

**MAS Reference link:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B9CAF3A19-5244-4FB1-8329-91DCE08BC85E%7D&file=MAS%201.4.3%20%E2%80%93%20Contrast%20(Minimum).docx&action=default&mobileredirect=true

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

![MAS1 4 3-Luminosity ratio for the comment and view button text is less than 4 5 1](https://user-images.githubusercontent.com/95913748/208601291-4941f153-d562-4560-a1b8-1b904c8ba072.png)
![MAS1 4 3-CCA is less than 4 5 1 for comment and view text](https://user-images.githubusercontent.com/95913748/208601298-752345de-d905-4a2b-a600-f97e7f5188e5.png)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Closing this issue as of now this issue is being tracked using below URL.

https://github.com/microsoft/AdaptiveCards/issues/8574
